### PR TITLE
bugfix: tmp folder path regarding windows

### LINKED ISSUE #187: No file change on the screen, despite recompile (RaydonZhu comment)
Hi
I face the same issue: 
RaydonZhu commented on 23 Nov 2016 • edited

I am trying to use this. everything works fine. However, it won't be able to 'recompile' automatically if file changes. For example, I change the text in a html file, and it seems that the app catched the changes and recompiled. However, i didn't see any changes on the screen. I had to rebuild it again, and then i could see the changes on the screen.
Why?
Thanks!

Thanks a lot in advance

### PR DESCRIPTION
**Pull request by Michael-xxxx**
See issue:
No file change on the screen, despite recompile (RaydonZhu comment) #187

**edited**
fix #187

### CODE REVIEW

File: config/gulp/tasks/component.js [hunk 1]
```diff
@@ -32,5 +32,7 @@ gulp.task('watch-css', function () {
 });
 
 function convertToTmpPath(file) {
-    return file.path.replace(/src\/app\//, config.tmpApp);
+	var re = new RegExp('src\' + path.sep + 'app\' + path.sep,"g");
```
Comment (antonybudianto) [inline config/gulp/tasks/component.js]: do you mean `'src\\'` and `'app\\'` ?

Comment (Michael-xxxx) [inline config/gulp/tasks/component.js]: No double slash will not work for windows.
Are there are issues with running under linux?
I see that the Tavis CI build failed but I didn't get details

Comment (Michael-xxxx) [inline config/gulp/tasks/component.js]: Ok now I found details regarding the build failure.....

Comment (Michael-xxxx) [inline config/gulp/tasks/component.js]: Sorry you are right I mean src\\ and app\\

Comment (Michael-xxxx) [inline config/gulp/tasks/component.js]: I just fixed the code

**Comment by antonybudianto** [comment]
@Michael-xxxx , nice, can you squash the commits into one?

**Comment by antonybudianto** [comment]
You can also create new PR to have 1 commit @Michael-xxxx

**Comment by Michael-xxxx** [comment]
I merge the two commits. I hope it is OK?

**Comment by antonybudianto** [comment]
@Michael-xxxx , no I mean, the commits should be one, currently it's 4 commits, can you open new PR with one commit? Thank you

**Pull request closed without merging**